config/default-spinnaker-local.yml

```diff
@@ -15,6 +15,19 @@ providers:
       # Store actual credentials in $HOME/.aws/credentials. See spinnaker.yml for
       # more information, including alternatives.
 
+  azure:
+    # If you want to deploy some services to Microsoft Azure (azure),
+    # set enabled and provide primary credentials for deploying.
+    # Enabling azure is independent of other providers.
+    enabled: false
+    defaultRegion: westus
+    primaryCredentials:
+      name: my-azure-account
+      clientId: my-client-id
+      appKey: my-app-key
```

Comment:
Same remark as for spinnaker.yml below, but if they are empty by default then you should add a comment indicating they need to be populated properly when enabling azure.